src/ostorlab/cli/rootcli.py

```diff
@@ -21,13 +22,20 @@
 )
 @click.option("-d", "--debug/--no-debug", help="Enable debug mode", default=False)
 @click.option("-v", "--verbose/--no-verbose", help="Enable verbose mode", default=False)
+@click.option(
+    "--gcp-logging-credential",
```

Comment:
Is there a reason why this is specific to GCP? Can we make the option generic, so we can support other cloud providers in the future?